caas/kubernetes/provider/bootstrap.go

```diff
@@ -1342,32 +1344,52 @@ func (c *controllerStack) buildContainerSpecForController(statefulset *apps.Stat
 		fmt.Sprintf("controller-%s", c.pcfg.ControllerId),
 		c.fileNameAgentConf,
 	)
-	var jujudCmd string
+	var jujudCmds []string
+	pushCMD := func(cmd string) {
```

Comment:
The `CMD` capitalisation looks a bit funny - can we make it `pushCmd`?
And `bootstrapStateCmd` and `machineCmd` below?